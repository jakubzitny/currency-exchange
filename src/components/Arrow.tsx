import React from 'react'

import styles from '../styles/Arrow.module.css'

export default function Arrow() {
  return (
    <div className={styles.container}>
      <div className={styles.arrow} />
    </div>
  )
}
