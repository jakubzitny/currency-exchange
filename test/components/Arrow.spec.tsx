import chai, { expect } from 'chai'
import chaiJestSnapshot from 'chai-jest-snapshot'
import { shallow } from 'enzyme'
import React from 'react'

import Arrow from '../../src/components/Arrow'

chai.use(chaiJestSnapshot)

describe('Arrow', () => {
  it('should render Arrow', () => {
    const arrow = <Arrow />
    const wrapper = shallow(arrow)

    expect(wrapper).to.matchSnapshot()
  })
})
