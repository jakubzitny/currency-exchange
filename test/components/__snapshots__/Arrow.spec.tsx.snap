// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow should render Arrow 1`] = `ShallowWrapper {}`;
